lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1073,6 +1073,12 @@ def cla(self):
             self.zaxis._set_scale(self._sharez.zaxis.get_scale())
         else:
             self.zaxis._set_scale('linear')
+            try:
+                self.set_zlim(0, 1)
+            except TypeError:
+                pass
+            except AttributeError:
```

Comment:
ok, I placed the 2D `cla()` at top, the mplot3d tests ran fine. Hopefully Travis won't complain.

yes it looks very long. I assume things like checks for shared axes are not relevant to mplot3d?
thanks for walking me through this!